src/css/toc.css

```
@@ -189,29 +235,42 @@
-  font-weight: 300;
+  top: 80px; /* Prototype: 80px */
+  max-height: calc(100vh - 100px); /* Prototype formula */
+  font-size: 13px; /* Prototype: 13px */
+  font-weight: 400;
   color: var(--toc-font-color);
   max-width: var(--toc-width);
-  overflow-y: auto;
-  overflow-x: hidden;
-  text-overflow: ellipsis;
   flex: 0 0 var(--toc-width);
   word-wrap: break-word;
```

Stylelint (17.11.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)